include/tcpdf_fonts.php

```diff
@@ -2001,7 +2001,7 @@ public static function UTF8StringToArray($str, $isunicode, &$currentfont) {
 		if ($isunicode) {
 			// requires PCRE unicode support turned on
 			$chars = TCPDF_STATIC::pregSplit('//','u', $str, -1, PREG_SPLIT_NO_EMPTY);
-			$carr = array_map(array('TCPDF_FONTS', 'uniord'), $chars);
+			$carr = array_map(array('TCPDF_FONTS', 'uniord'), is_array($chars) ? $chars : array());
```

Comment:
Can you provide an example code to trigger this?

Reply:
I think the problem is related to https://github.com/tecnickcom/TCPDF/issues/470
When providing a non utf8 string the `preg_split` might return `false`. causing the array_map to throw a warning or error. But I'm not sure if this is the optimal solution, as it would simply remove the text from the pdf...